### PR TITLE
Add runner for local environment using `testcontainers`

### PR DESCRIPTION
**Pull request by dmytro-lazebnyi**
Execute `./run_local_environment.sh` to start the whole infrastructure of configuration service (using Docker containers) on the local PC.

### CODE REVIEW

File: scripts/run_local_environment.sh [hunk 1]
```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+
+SCRIPTS_DIR=$(dirname $0)
+TARGET_DIR="${SCRIPTS_DIR%/*}"/scalecube-configuration-benchmarks/target
+JAR_FILE=$(ls ${TARGET_DIR} | grep -m 1 .jar)
+
```
Comment (segabriel) [inline scripts/run_local_environment.sh:12]: maybe let's add a check on jar existence and if it will false run `mvn clean install -DskipTests`

**Review by segabriel — DISMISSED** [review]

File: scripts/run_local_environment.sh (hunk 1 above)
Comment (dmytro-lazebnyi) [reply scripts/run_local_environment.sh:12]: Added.

**Review by segabriel — DISMISSED** [review]

File: scripts/run_local_environment.sh [hunk 2]
```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+
+SCRIPTS_DIR=$(dirname $0)
+PROJECT_DIR="${SCRIPTS_DIR%/*}"
+TARGET_DIR=${PROJECT_DIR}/scalecube-configuration-benchmarks/target
+JAR_FILE=$(ls ${TARGET_DIR} | grep -m 1 .jar)
```
Comment (segabriel) [inline scripts/run_local_environment.sh]: let's grep by `SNAPSHOT.jar` my system always retrieves `..-docker.info.jar`

Comment (dmytro-lazebnyi) [reply scripts/run_local_environment.sh]: Done.

**Review by segabriel — APPROVED** [review]

**Pull request merged**